src/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -84,17 +84,14 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.Normalization.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.Normalization.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs">
+    <Compile Include="$(CommonPath)\Interop\Windows\normaliz\Interop.Idna.cs" Condition="'$(TargetGroup)'=='uap101aot'">
```

Comment:
Why do you need to keep this when you are adding it to the ApiCompatBaseline.uap101aot.txt  baseline anyway?